code/Player/PropPossession.cs

```diff
@@ -0,0 +1,207 @@
+﻿using System;
+using Sandbox;
+using TTT.UI;
+
+namespace TTT;
+
+public partial class PropPossession : EntityComponent<Player>
+{
+	[Net]
+	public Prop Prop { get; set; }
```

Comment:
```suggestion
	public Prop Prop { get; set; }

```
Add a space inbetween these